coalib/misc/Constants.py

```diff
@@ -13,6 +13,11 @@
                  "happen. When asked for, the following information "
                  "may help investigating:")
 
+VERSION_CONFLICT_MESSAGE = ("There is a conflict in the version of a package "
+                            "you have installed and the requirements of "
+                            "coala. The following information may help "
+                            "to resolve this conflict: ")
```

Comment:
we need a smart solution there, cleanest thing for the user would be a
virtualenv, alternative force overwriting some versions or something I
think we should mention both

Sincerely,

Lasse Schuirmann

contact@viperdev.io
http://viperdev.io/

2016-03-16 19:14 GMT+01:00 Adhityaa Chandrasekar <notifications@github.com>:

> In coalib/misc/Constants.py
> <https://github.com/coala-analyzer/coala/pull/1894#discussion_r56386878>:
>
> > @@ -13,6 +13,11 @@
> >                   "happen. When asked for, the following information "
> >                   "may help investigating:")
> >
> > +VERSION_CONFLICT_MESSAGE = ("There is a conflict in the version of a package "
> > +                            "you have installed and the requirements of "
> > +                            "coala. The following information may help "
> > +                            "to resolve this conflict: ")
>
> Ah, I think we can access the requirement package (with correct version).
> I wonder if pip3 install <package_name> would work for the string
> returned by pkg_resources.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1894/files/d308fec94405cafe82a91c200bd0037e3c681b2c#r56386878>
>